test/runner/lib/util.py

```diff
@@ -89,13 +89,14 @@ def run_command(args, cmd, capture=False, env=None, data=None, cwd=None, always=
     :type always: bool
     :type stdin: file | None
     :type stdout: file | None
+    :type cmd_verbosity: int | 1
```

Comment:
Only the type is needed here, not the default value, so just `:type cmd_verbosity: int` is needed.